features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/documentation/download.jag

```diff
@@ -72,8 +72,10 @@ if (resourceUrl.indexOf("/documentation/files") != -1 ||
 if (object == null) {
     response.status = 500;
 } else {
+    var name = object.name;
+    name = name.split('--');
     response.contentType = "application/force-download";
-    response.addHeader("Content-Disposition", "attachment; filename = \"" + object.name + "\"");
+    response.addHeader("Content-Disposition", "attachment; filename = \"" + name[1] + "\"");
```

Comment:
can be done in a single line 
`response.addHeader("Content-Disposition", "attachment; filename = \"" + object.name.split('--')[1] + "\"");`

Reply:
Fixed with 7e037aab0a18987b058d1b8b953be34c43303821